extension/src/plots/webview/contract.ts

```diff
@@ -102,6 +102,8 @@ export type CustomPlotsData = {
   nbItemsPerRow: number
   enablePlotCreation: boolean
   height: PlotHeight
+  hasUnfilteredExperiments: boolean
+  hasAddedPlots: boolean
```

Comment:
We need to add `hasAddedPlots` because `plots` will be an empty array if there is no table data OR no added plots. 

Another option is to have `plots` only be empty when there are no added plots and fill `plots` with technically broken plots when there is no table data. These broken plots wouldn't be rendered of course since we don't render any plots when `hasUnfilteredExperiments` is false. I chose `hasAddedPlots` since I felt like that solution has more clarity.